desktop/apps/artsy_primer/personalize/client/index.coffee

```diff
@@ -76,10 +77,15 @@ module.exports.PersonalizeRouter = class PersonalizeRouter extends Backbone.Rout
 
     @$el.attr 'data-state', 'loading'
 
-    $.when.apply(null, [
-      @user.save()
-      $.post('/flash', message: 'Thank you. Please expect your personalized portfolio in the next 2 business days.')
-    ]).always =>
+    Q.all [
+      $.post('/artsy-primer/set-sailthru')
+    ]
+    .then =>
+      Q.all [
+        @user.save()
```

Comment:
if we want, we can also pass in `receive_personalized_email: false` thing too. I still don't understand _why_ we need to do that if we can just disable the emails via the `artsy_primer` var in ST, but at least their profile preferences will be correct?